app/frontend/src/components/AnalysisPanel/ThoughtProcess.tsx

```diff
@@ -12,9 +12,9 @@ interface Props {
 export const ThoughtProcess = ({ thoughts }: Props) => {
     return (
         <ul className={styles.tList}>
-            {thoughts.map(t => {
+            {thoughts.map((t, ind) => {
                 return (
-                    <li className={styles.tListItem}>
+                    <li className={styles.tListItem} key={ind}>
```

Comment:
We were getting a bunch of React errors locally from missing key= values. An index isnt the best thing to use but it should be okay for our situation.